centreon/tests/e2e/cypress/e2e/Dashboards/04-dashboard-deletion.feature

```diff
@@ -1,15 +1,18 @@
-Feature: Deleting an existing dashboard
+@REQ_MON-18494
+Feature: Manage dashboards (create/delete)
```

Comment:
CF comment above, let's create another ticket for the deletion in the same epic as the creation